src/components/Router/PrivateRoute.jsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import { connect } from 'react-redux';
+import { Route, Redirect } from 'react-router-dom';
+
+// eslint-disable-next-line react/prop-types
+const PrivateRoute = ({ component: Component, ...props }) => {
+  return (
+    <Route
+      {...props}
+      render={
+        matchProps =>
+          localStorage.getItem('token') ? (
+            <Component {...matchProps} />
+          ) : (
```

Comment:
We need to plug this to "isLoggedIn" in redux store and not directly to local storage, what do you think?